Extension/src/common.ts

```diff
@@ -573,6 +576,7 @@ export async function checkInstallBinariesExist(): Promise<boolean> {
                 if (!await checkFileExists(file)) {
                     installBinariesExist = false;
                     console.log(`Extension file ${file} is missing.`);
+                    Telemetry.logLanguageServerEvent("missingBinary", { "source": `${file}` });
```

Comment:
You should use `path.basename(file)` instead. We can't log absolute paths in telemetry.

Reply:
fixed.